pkgs/development/tools/vdexExtractor/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, fetchFromGitHub, stdenv, zlib }:
+stdenv.mkDerivation rec {
+  pname = "vdexExtractor";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "anestisb";
+    repo = "vdexExtractor";
+    rev = version;
+    sha256 = "sha256-dSzNK1dvllmY9HcA7u0tbhDM8tY/I6GqzgAJ56mB5vE=";
+  };
+
+  buildPhase = ''
+    bash ./make.sh
+  '';
+
+  installPhase = ''
+    install -D bin/vdexExtractor $out/bin/vdexExtractor
+  '';
+
+  nativeBuildInputs = [
```

Comment:
zlib is used as a library, right? Which means it goes to buildInputs.

Reply:
Ah yes, absolutely.